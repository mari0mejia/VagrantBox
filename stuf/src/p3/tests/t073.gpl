// use std::numbers::pi from #include<numbers> (as of C++20)
// DO NOT CREATE YOUR OWN VALUE FOR PI
double d = sin( 7 );
double e = sin( 12.5 );
string g = sin( 7 );
string h = sin( 12.5 );
