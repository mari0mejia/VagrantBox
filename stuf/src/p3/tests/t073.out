gpl.cpp::main()
  input file(tests/t073.gpl)
  random seed(42)
  read_keypresses_from_standard_input(false)
  dump_pixels(false)
  symbol_table(true)
  graphics(false)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().

No errors found (parser probably worked correctly).


Printing the symbol table from main()
double d = 0.121869
double e = 0.21644
string g = "0.121869"
string h = "0.216440"
Graphics is turned off by the Makefile.  Program exiting.

gpl.cpp::main() done.
